python_modules/dagster/dagster/core/system_config/composite_descent.py

```diff
@@ -124,7 +124,7 @@ def _composite_descent(parent_stack, solids_config_dict, resource_defs, is_using
             )
             if not config_mapped_solid_config.success:
                 raise DagsterInvalidConfigError(
-                    f"Error in config for {solid.describe_node()()}".format(solid.name),
+                    f"Error in config for {solid.describe_node()}".format(solid.name),
```

Comment:
I think a user recently filed a bug report for this. Mind factoring it out into a separate PR?

Reply:
forgot I did this lol